### ISSUE TITLE
Wrong exception caught?

### ISSUE THREAD

**Issue by alexpdp7** [ihabunek/toot#504]
https://github.com/ihabunek/toot/blob/master/toot/tui/app.py#L366

I'm using Takahe and when:

```
self.account = api.find_account(self.app, self.user, acct)
```

runs, I get:

```
File "/home/alex/.local/pipx/venvs/toot/lib/python3.11/site-packages/toot/api.py", line 38, in find_account
raise ConsoleError("Account not found")
```

, which messes up my display. If I broaden the exception caught, things seem to work.

**Comment by alexpdp7** [comment]
https://github.com/alexpdp7/toot/commit/01873f17d96dc094d7b460a0c9048c2e221b13cc as a demo that patches the issue for me. Perhaps this is correct and you be merged. (Feel free to cherry-pick or whatever.)

**Comment by ihabunek** [comment]
Thanks. Fixed and released in [0.47.0](https://github.com/ihabunek/toot/releases/tag/0.47.0).
